js/repl/Repl.js

```diff
@@ -71,6 +71,7 @@ type State = {
   pluginSearch: ?string,
   showOfficialExternalPlugins: boolean,
   loadingExternalPlugins: boolean,
+  filename: ?string,
```

Comment:
You also need to add the setting to the state initializer similar to https://github.com/babel/website/pull/1676/files#diff-13734ac24709204bbae97c4b659d1ba1R149